### PR TITLE
test: add real tests for 3 empty/stub templates (batch 2)

### PR DESCRIPTION
**Pull request by Aydin-ab**
Batch 2 — three templates whose tests were empty / a 2-line stub / a TODO+echo no-op. Notebook-driven ones **strip `skip-in-ci` cells (nbconvert) + execute (papermill --log-output)**, local-only.

- **getting-started** (was 0 bytes): run the intro notebook (CI-safe Ray hello-world).
- **serve-stable-diffusion-v2** (was a 2-line stub): the notebook documents `serve run` only as a terminal step, so the test drives `main.py` + `query.py` directly — `serve run main:stable_diffusion_app` + retry `query.py` + assert PNG + `serve shutdown`. (No notebook execution here.) Skips `anyscale service deploy`.
- **vhol-ray-data** (was pip+TODO+echo): run the full Ray Core + Ray Data lab (SDXL-turbo + Qwen2.5-0.5B). 0 tags.

Unverified — staging provisioning down (same as #708). `serve-stable-diffusion-v2` timeout (900s) is tight against the SDXL download; bump if the first run times out.

### CODE REVIEW

File: tests/serve-stable-diffusion-v2/tests.sh [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+
+# First request blocks through GPU provisioning + SDXL load, so retry the query.
+ok=false
+for i in $(seq 1 5); do
+  python query.py || true
+  if python -c "
+import os
+from PIL import Image
+assert os.path.getsize('image.png') > 1000
+Image.open('image.png').verify()
```
[the remark this answers is not in the document]
Comment (Aydin-ab) [reply tests/serve-stable-diffusion-v2/tests.sh:19]: Verified this is a false positive — the `python -c` body is at column 0 (no leading indentation), so it parses without an `IndentationError`. Confirmed by running the exact snippet: it fails only on a missing import locally, not a parse error, and PIL ships with the template's diffusers/transformers deps in CI. The readiness-retry loop works as intended.

**Comment by Aydin-ab** [comment]
/test-template getting-started serve-stable-diffusion-v2 vhol-ray-data

**Comment by Aydin-ab** [comment]
/test-template getting-started serve-stable-diffusion-v2 vhol-ray-data

**Review by elliot-barn — APPROVED** [review]
